controllers/news-controllers.errors.js

```diff
@@ -1,9 +1,11 @@
-exports.invalidPath = (req, res) => {
+exports.invalidPath = (req, res) => 
+  res.status(404).send({msg: "Invalid path."});
 
-  res.status(404).send({msg: "Invalid path"});
+exports.handlePSQLerrors = (err, req, res, next) =>
+  /22P02/.test(err.code) ? res.status(400).send({msg: "Bad request."}) : next(err);
```

Comment:
Thinking ahead, it might be better to use if statements, as you're going to have lots of psql codes to check for different things. At the moment, you'll end up with nested if statements to check them all. 
You can also do err.code === '22P02'